runtime/flangrti/trace_lin.c

```diff
@@ -264,5 +269,91 @@ __abort_sig_init(void)
     n++;
   }
 }
+#else
 
```

Comment:
Delete this blank line to maintain a consistent style.

Reply:
```suggestion
```